public/app/core/logs_model.ts

```diff
@@ -312,7 +312,7 @@ export function logSeriesToLogsModel(logSeries: DataFrame[]): LogsModel | undefi
       const searchWords = series.meta && series.meta.searchWords ? series.meta.searchWords : [];
 
       let logLevel = LogLevel.unknown;
-      if (logLevelField) {
+      if (logLevelField && logLevelField.values.get(j)) {
```

Comment:
This blowed up in same cases because we weren't actually mapping the field properly, but feels like this should be here anyway because we cannot be sure what we get from `values.get(j)` on a Field.

Reply:
Could simplify a bit with
```suggestion
      if (logLevelField?.values.get(j)) {
```